source/extensions/compression/zstd/common/base.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <memory>
+
+#include "envoy/buffer/buffer.h"
+
+#include "zstd.h"
+
+#define UNKNOWN_DICTIONARY_ID -1U
```

Comment:
This definition is unused, how it works?

Reply:
Thanks! Fix now